test/dialect/postgresql/test_dialect.py

```diff
@@ -1,8 +1,11 @@
+from contextlib import nullcontext
 import dataclasses
 import datetime
 import logging
 import logging.handlers
 
+import pytest
```

Comment:
**mike bayer** (zzzeek) wrote:

remove this...

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4755